content/en/blog/_posts/2023-12-14-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,210 @@
+Kubernetes and as such they began their life as external cloud controller
+managers.
+
+### Upgrading from older Kubernetes versions
+
+If you are upgrading from a Kubernetes release older than 1.26, and you are on
+AWS, Azure, GCE, OpenStack, or vSphere then you will need to enable the
+`--cloud-provider=external` flag, and follow the advice for installing and
+running a cloud controller manager for your provider.
+
+Please read the documentation for
+[Cloud Controller Manager Administration][ccmadmin] and
+[Migrate Replicated Control Plane To Use Cloud Controller Manager][ccmha]. See
+below for links to specific cloud provider implementations.
+
+## Where to find a cloud controller manager?
+
+At its core, this announcement is about the cloud provider integrations that
+were previously included in Kubernetes. As these components move out of the
+core Kubernetes code and into their own repositories, it is important to note
+a few things:
+
+First, SIG Cloud Provider maintains a reference framework for developers who
+wish to create cloud controller managers for any provider. See the
+[cloud-provider repository][cloud-provider] for more information about how
+these controllers work and how to get started creating your own.
+
+Second, there are many cloud controller managers available for Kubernetes.
+This post is addressing the providers that have been historically included
+with Kubernetes but are now in the process of being removed. If you need a
+cloud controller manager for your provider and do not see it listed here,
+please reach out to the cloud provider you are integrating with or the
+[Kubernetes SIG Cloud Provider community][sig] for help and advice. It is
+worth noting that while most cloud controller managers are open source today,
+this may not always be the case. Users should always contact their cloud
+provider to learn if there are preferred solutions to utilize on their
+infrastructure.
+
+### Cloud provider integrations provided by the Kubernetes project {#cloud-provider-integrations}
+
+* AWS - https://github.com/kubernetes/cloud-provider-aws
+* Azure - https://github.com/kubernetes-sigs/cloud-provider-azure
+* GCE - https://github.com/kubernetes/cloud-provider-gcp
+* OpenStack - https://github.com/kubernetes/cloud-provider-openstack
+* vSphere - https://github.com/kubernetes/cloud-provider-vsphere
+
+If you are looking for an automated approach to installing cloud controller
+managers in your clusters, the [kOps][kops] project provides a convenient
+solution for managing production-ready clusters.
+
+## Want to learn more?
+
+Cloud providers and cloud controller managers serve a core function in
+Kubernetes. Cloud providers are often the substrate upon which Kubernetes is
+operated, and the cloud controller managers supply the essential lifeline
+between Kubernetes clusters and their physical infrastructure.
+
+This post exposes one aspect of how the Kubernetes community interacts with
```

Comment:
```suggestion
This post covers one aspect of how the Kubernetes community interacts with
```

"exposes" is an odd choice, because it wasn't actually _hidden_?

Reply:
i don't agree with the characterization, but i've changed it.

Reply:
Both uses can work, but when writing for a broad international audience I always think about reducing ambiguity wherever feasible.